mailpoet/assets/js/src/newsletters/newsletters.jsx

```diff
@@ -38,6 +38,7 @@ import { CorruptEmailNotice } from '../notices/corrupt-email-notice';
 import { LegacyAutomaticEmailsNotice } from '../notices/legacy-automatic-emails-notice';
 import { TopBarWithBeamer } from '../common/top-bar/top-bar';
 import { BackButton, PageHeader } from '../common/page-header';
+import { ExampleApp } from './test-app';
```

Comment:
Seems like test code :D 
Are you extending the current PR with the table as well?